server/src/worker/distribution-worker.ts

```diff
@@ -1,20 +1,19 @@
 import { App } from '../core';
+import { CronJob } from 'cron';
 
-export function runDistributionWorker(app: App, intervalMilliseconds: number) {
-  async function workLoop() {
-    try { 
-      console.log(`Starting distribution process at ${new Date()}...`);
-      const result = await app.donationDistributions.distributeDonations();
-      console.log(`Completed distribution process at ${new Date()}`);
-      console.log(result);
-      console.log();
-      setTimeout(workLoop, intervalMilliseconds);
-    }
-    catch(e) {
-      console.error(e);
-      setTimeout(workLoop, intervalMilliseconds);
-    }
-  }
+export function runDistributionWorker(app: App, intervalMinutes: number) {
+  const job = new CronJob(`0 */${intervalMinutes} * * * *`, async () => {
```

Comment:
Let's not assume that the interval will be in minutes. We want to be able to have better control of the interval (e.g. we might decide that it should run at 2PM every day). For that reason, we should be able to pass the interval in the cron syntax.

e.g. 
```ts
function runDistributionWorker(app: App, interval: string) {
  const job = new CronJob(interval, ...);
}
```